static/docs/commands-reference/fetch.md

```diff
@@ -1,62 +1,243 @@
 # fetch
 
-Fetches data files from the remote storage to the local cache.
+Get files that are under DVC control from remote storage into the local cache.
+Fetching means downloading from the
+[remote](https://dvc.org/doc/commands-reference/remote#description) unless the
+files in question already exist in the local cache, in which case nothing is
+done.
 
-The set of data files to fetch (usually it means downloading from the remote
-storage) is determined by analyzing all `.dvc` files in the current branch,
-unless `--all-branches` is specified.
-
-The command fetches only outputs of a specific stage if dvc file is specified
-`dvc push data.zip.dvc`.
-
-See `dvc remote`, `dvc config` and 
-[remote storages](https://dvc.org/doc/get-started/configure)
-for more information on how to configure the remote storage.
+## Synopsis
 
 ```usage
-    usage: dvc fetch [-h] [-q] [-v] [-j JOBS] [-r REMOTE] [-a]
-                     [-T] [-d]
+    usage: dvc fetch [-h] [-q | -v] [-j JOBS]
+                     [--show-checksums] [-r REMOTE]
+                     [-a] [-T] [-d] [-R]
                      [targets [targets ...]]
-
+    
+    Fetch data files from the cloud.
+    
     positional arguments:
-      targets               DVC files.
+      targets               DVC files (stages).
+```
+
+## Description
+
+When checking out an existing project repository, files under DVC control should
+exist in a remote, but won't be in your local machine. (Refer to `dvc remote`
+for more information on DVC remotes.) These necessary data or model files
+are be listed as dependencies or outputs in a DVC file (target stage) however.
+(See [DVC File Format](https://dvc.org/doc/user-guide/dvc-file-format) for  more
+information on dependencies and outputs.)
+
+`dvc fetch` ensures that the files needed for a DVC file to be
+[reproduced](/doc/get-started/reproduce) exist in the local cache. If no
+`targets` are specified, the set of data files to fetch is determined by
```

Comment:
Not clear what's happening if targets are specified. Why don't we adopt similar description from push or pull?

Reply:
OK, I will make it similar to pull and push. I was previously asked to start the command description with an explanation of the context of remotes etc.

Reply:
Updated